src/app/settings/WishListSettings.tsx

```diff
@@ -94,31 +124,36 @@ class WishListSettings extends React.Component<Props> {
     );
   }
 
+  private wishListUpdateEvent = () => {
+    let wishListSource = this.state.wishListSource;
+
+    if (!isUri(wishListSource)) {
+      alert(t('WishListRoll.InvalidExternalSource'));
+      return;
+    }
+
+    wishListSource = wishListSource?.trim();
+
+    if (this.props.wishListSource === wishListSource) {
+      return;
+    }
+
+    this.props.setSetting('wishListSource', wishListSource);
+
+    store.dispatch(fetchWishList(true));
+
+    refresh();
```

Comment:
What's the refresh for? You won't have actually loaded the new wish list at this point.